recipe/conda_build_config.yaml

```diff
@@ -26,9 +26,6 @@ cuda_compiler:  # [linux64 or win or (linux and ppc64le)]
 cuda_compiler_version:
```

Comment:
I think we can clear this file now.

Everything under `# When adding or removing cuda versions, make sure that the following entries are "zipped";` can be removed so we go back to conda-forge's default config after a rerender.

Reply:
We might need to keep the ppc64le parts though.

Reply:
Can you tell me exactly what I need to change?  I don't completely understand.